rpi_backlight.py

```diff
@@ -86,49 +102,66 @@ def set_brightness(value: int, smooth: bool = False, duration: float = 1) -> Non
         diff = abs(value - actual)
         while actual != value:
             actual = actual - 1 if actual > value else actual + 1
-
-            _set_value("brightness", actual)
+            set_brightness_value(actual)
             time.sleep(duration / diff)
     else:
-        _set_value("brightness", value)
+        set_brightness_value(value)
 
 
 def set_power(on: bool) -> None:
     """Set the display power on or off.
-
     :param on: Boolean whether the display should be powered on or not
     """
-
     # 0 is on, 1 is off
-    _set_value("bl_power", int(not on))
+    if MODE == "MODE_RPI":
+        _set_value("bl_power", int(not get_power))
+    elif MODE == "MODE_TINKERBOARD":
+        if on:
+            value = 255
+        else: 
+            value = 0
+        set_brightness_value(value)
 
 
+def toggle_power() -> None:
+    """Toggle the display power on or off."""
+    set_power(not get_power())
+
 def _create_argument_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(
         description="Control power and brightness of the "
         'official Raspberry Pi 7" touch display.'
     )
+
     parser.add_argument(
         "-b",
         "--brightness",
         metavar="VALUE",
         type=int,
-        choices=range(11, 256),
-        help="set the display brightness to VALUE (11-255)",
+        choices=range(0, 256),
+        help="set the display brightness to VALUE (0-255)",
     )
+
     parser.add_argument(
         "-d", "--duration", type=int, default=1, help="fading duration in seconds"
     )
     parser.add_argument(
         "-s",
         "--smooth",
+        default=False,
```

Comment:
Not required because of `action="store_true"`

Reply:
Why does storing true do the same as defaulting to false?

Reply:
Ohh nevermind you wanted defaulted to true

Reply:
Fixed